rules/feature/targettype.yaml

```diff
@@ -96,3 +96,11 @@
 		RequiresForceFire: true
 	Targetable@TargetTypeNoAutoTarget:
 		TargetTypes: NoAutoTarget
+
+^FeatureTargettypeFloraAir:
```

Comment:
Well I did not know about this, I thought it would crash if you have none selected.